statsmodels/tsa/vector_ar/tests/test_var.py

```diff
@@ -15,6 +15,18 @@
 import numpy as np
 import pytest
 
+have_matplotlib = False
+try:
+    import matplotlib
+    have_matplotlib = True
+    import matplotlib.pyplot as plt
+    plt.switch_backend('Agg')  # otherwise segfault on windows
```

Comment:
Good to know about the windows segfault.  Also prevents exceptions (I forget which kind) on headless Linux.